documentation/puppetdb_connection.markdown

```diff
@@ -48,9 +48,25 @@ You can use a comma separated list of URLs if there are multiple PuppetDB instan
 
     server_urls = https://puppetdb1.example.com:8081,https://puppetdb2.example.com:8081
 
-The PuppetDB terminus will always attempt to connect to the first PuppetDB instance specified (listed above as puppetdb1). If a server-side exception occurs, or the request takes too long (see [`server_url_timeout`](#server_url_timeout)), the PuppetDB terminus will attempt the same operation on the next instance in the list.
+The default value is `https://puppetdb:8081`.
 
-The default value is https://puppetdb:8081
+For read operations, the PuppetDB terminus will always attempt to connect to the first PuppetDB instance specified (listed above as puppetdb1). If a server-side exception occurs, or the request takes too long (see [`server_url_timeout`](#server_url_timeout)), the PuppetDB terminus will attempt the same operation on the next instance in the list.
+
+For write operations, all commands are sent to all PuppetDB instances in the list. 
+
+Configurations with multiple `server_urls`, along with the `min_successful_writes` and `write_only_server_urls` settings, can be used with the Puppet Enterprise version of PuppetDB to set up a high-availability PuppetDB deployment. Please note that, absent the synchronization support in Puppet Enterprise, multiple PuppetDB instances can easily diverge from each other due to normal transient network issues. We do not recommend using these settings outside of a supported high-availability deployment. `
+
+### `min_successful_writes`
+
+When writing data (submitting commands) to PuppetDB, this is the minimum number of machines to which the command must be successfully sent to consider the write successful. The default value is 1, which should be appropriate for most single- or dual-PuppetDB deployments. You may wish to change this value if you have particularly high data durability requirements. For example, if you want to ensure that you can withstand the loss of a node even when running PuppetDB in a degraded state, you could deploy a cluster of a 3 PuppetDB instances and set `min_successful_writes` to 2. 
+
+### `write_only_server_urls`
```

Comment:
Yes, it does exactly what it says in both foss and PE. Even in foss, you can use it to set up a second server and send all your data to it for testing.